src/microbe_stage/editor/TolerancesEditorSubComponent.cs

```diff
@@ -903,6 +918,20 @@ private void UpdateCurrentValueDisplays()
         {
             uvResistanceModifierLabel.Visible = false;
         }
+
+        // Update markers
+        temperatureToleranceMarker.OptimalValue = patchTemperature / 100.0f;
+
+        minPressureToleranceMarker.OptimalValue = patchPressure / 70000000;
+        maxPressureToleranceMarker.OptimalValue = patchPressure / 70000000;
```

Comment:
I now like the approach to the markers. But I still don't like that the pressure has both the marker on the min and max sliders. Isn't that more confusing than if the marker was only on the upper pressure slider? I could very well be wrong in which is the more understandable option. What's your reasoning for adding the optimal marker for both pressure sliders?